index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Linkbrary</title>
+
+
+  <meta property="og:title" content="Linkbrary" key="title" />
+  <meta
+    name="description"
+    property="og:description"
+    content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+    key="description"
+  />
+  <meta
+    property="og:image"
+    content="https://visitbusan.net/uploadImgs/files/cntnts/20211130150754165_wufrotr"
+    key="image"
+  />
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:site" content="@linkbrary" />
+  <meta name="twitter:title" content="Linkbrary" />
+  <meta name="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+  <meta
+    name="twitter:image"
+    content="https://visitbusan.net/uploadImgs/files/cntnts/20211130150754165_wufrotr"
+  />
+  <link
+    rel="stylesheet"
+    type="text/css"
+    href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+  />
+
+
+  <link rel="stylesheet" href="src/style.css">
+</head>
+<body>
+  <nav>
+    <div class="nav">
+      <a href="index.html">
+        <img src="image/logo.svg" alt="logo" class="logo">
+      </a>
+      <a class="button Login_button" href="signin.html">로그인</a>
+    </div>
+  </nav>
+  <header>
+    <section class="main_section">
+      <h1><span>세상의 모든 정보</span>를<br> 쉽게 저장하고 <br class="hidden_pc">관리해 보세요</h1>
+      <a class="button link_add_button" href="signup.html">링크 추가하기</a>
+      <img class="exam_img" src="image/hero.png">
+    </section>
+  </header>
+  <article>
+    <section class="back_white">
+      <div class="contents">
+        <div>
+          <h2><span class="subtitle1">원하는 링크</span>를<br class="hidden_mobile">
+            저장하세요</h2>
+          <span class="subtext">나중에 읽고 싶은 글, 다시 보고 싶은 영상,<br class="hidden_mobile">
```

Comment:
@JeongSabibi 

[수정 요청]

태그에 닫힘 태그가 없네요? 혹시 prettier 설정이 되어 있으신가요?
vscode 를 사용하신다면 extension 에서 설치 하실수 있습니다.

```suggestion
          <h2><span class="subtitle1">원하는 링크</span>를<br class="hidden_mobile" />
            저장하세요</h2>
          <span class="subtext">나중에 읽고 싶은 글, 다시 보고 싶은 영상,<br class="hidden_mobile" / >
```


<img width="1030" alt="image" src="https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/assets/56149367/e6effaac-7751-43c6-9371-6de304e56968">


유용한 extension 이 많습니다. 검색 해보시면 여러가지 있는걸 알수 있어요.

Reply:
@JeongSabibi 

[질문]
prettier 가 무엇일까요? 왜 코드 포맷을 맞출까요? 하나의 레포에서 많은 사람들이 포맷을 맞추지 않고 작업을 하게 되면 어떤 상황이 발생할까요?

- https://prettier.io/
- https://velog.io/@treejy/React-ESLint%EC%99%80-Prettier%EA%B0%80-%EB%AC%B4%EC%97%87%EC%9D%B4%EB%A9%B0-%EC%99%9C-%ED%95%84%EC%9A%94%ED%95%98%EA%B3%A0-%EC%96%B4%EB%96%BB%EA%B2%8C-%EC%82%AC%EC%9A%A9%ED%95%98%EB%8A%94%EC%A7%80